lodash/lodash.d.ts

```diff
@@ -10445,6 +10446,13 @@ declare module _ {
         isEmpty(): boolean;
     }
 
+    interface LoDashExplicitWrapperBase<T,TWrapper> {
+        /**
+         * @see _.isEmpty
+         */
+        isEmpty(): LoDashExplicitWrapper<boolean>;
```

Comment:
You're absolutely right, my bad. Doesn't make much sense to me, but the return type is a wrapper indeed :+1: